examples/wasm-shell/package.json

```diff
@@ -11,24 +11,24 @@
     "type": "git",
     "url": "git+https://github.com/wasmerio/wasmer-js.git"
   },
-  "author": "Aaron Turner",
+  "author": "Wasmer Engineering Team <engineering@wasmer.io>",
```

Comment:
Oh oops, thanks for catching that! 😄 Didn't know we had an official Author thing. I'd seen "Syrus Akbary", so I thought we were just putting our names 😂